packages/api/src/external/carequality/document/download-documents.ts

```diff
@@ -1,3 +0,0 @@
-// FILE TO IMPLEMENT DOC RETRIEVAL FROM CQ
```

Comment:
This was leftover from before. Its now the `process-document-retrieval-results.ts`